Include way to set Map yaml file path for mapserver node behavior to migration guide (#366)
ros-planning/navigation.ros.org: migration/Humble.rst

***********************

Adding a new smoother algorithm, the Savitzky-Golay smoother to the smoother server plugin list. See the configuration guide :ref:`configuring_savitzky_golay_filter_smoother` for more details.

> Set Map yaml file path for mapserver node

Reviewer: ```suggestion
Changes to Map yaml file path for map_server node in Launch
```

> ***********************

Reviewer: The `***` line has to be the same length as the title for it to render properly

> `PR #3174 <https://github.com/ros-planning/navigation2/pull/3174>`_ adds a way to set the path to map yaml file for the mapserver node either from the yaml file or using the launch configuration parameter ``map`` giving priority to the launch configuration parameter.

Reviewer: ```suggestion
`PR #3174 <https://github.com/ros-planning/navigation2/pull/3174>`_ adds a way to set the path to map yaml file for the map_server node either from the yaml file or using the launch configuration parameter ``map`` giving priority to the launch configuration parameter.
```

> `PR #3174 <https://github.com/ros-planning/navigation2/pull/3174>`_ adds a way to set the path to map yaml file for the map_server node either from the yaml file or using the launch configuration parameter ``map`` giving priority to the launch configuration parameter.

Reviewer: ```suggestion
`PR #3174 <https://github.com/ros-planning/navigation2/pull/3174>`_ adds a way to set the path to map yaml file for the map_server node either from the yaml file or using the launch configuration parameter ``map`` giving priority to the launch configuration parameter. ``yaml_filename`` is no longer strictly required to be present in ``nav2_params.yaml``.
```
